### DISCUSSION TITLE
文章页面信息的自定义

### DISCUSSION THREAD

**Discussion by Lzkkuan** [theme-shoka-x/hexo-theme-shokaX#290] [General]
大佬们好，我是一个刚跟着网站的指引配置好shokaX的新手，想问一下像这种文章页面的格式在哪里可以修改

![e04f46e102b54e672dabb64295ca7d0](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/141710483/987014e2-b42c-47af-bafc-be12e154ac86)

![f25623f05322164bbdaa9f2dcc65aec](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/141710483/7e849873-6655-41aa-83c6-7f4278c8815b)

以及这个：

![1b7613cfe4d48c9b5c81084ea892472](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/141710483/5648622e-f3c1-4e17-930c-203cc8b7021e)

感谢大佬

**Comment by D-Sketon** [comment]
首先修改博客语言，其次如果对翻译不满意可以查看https://github.com/theme-shoka-x/hexo-theme-shokaX/blob/main/languages%2FREADME.md 进行修改

**Reply by Lzkkuan** [reply]
好的谢谢，我去试试
